app/Http/Controllers/WikiController.php

```diff
@@ -124,7 +125,15 @@ public function create(Request $request): \Illuminate\Http\Response
 
         // dispatch elasticsearch init job to enable the feature
         if ( Config::get('wbstack.elasticsearch_enabled_by_default') ) {
-            dispatch(new ElasticSearchIndexInit($wiki->id));
+            $clusterToIndex = Config::get('wbstack.elasticsearch_cluster_to_index');
+            $sharedIndex = Config::get('wbstack.elasticsearch_shared_index');
+
+            if ( $clusterToIndex ) {
```

Comment:
Is this running every wiki creation time a problem (i.e. trying to recreate the shared index) a problem?